rules_building_block/defense_evasion_masquerading_windows_dll.toml

```
@@ -40,9 +40,24 @@ library where event.action == "load" and dll.Ext.relative_file_creation_time <=
         "?:\\$WINDOWS.~BT\\Work\\*",
         "?:\\Windows\\WinSxS\\*",
         "?:\\Windows\\SoftwareDistribution\\Download\\*",
-        "?:\\Windows\\assembly\\NativeImages_v*"
+        "?:\\Windows\\assembly\\NativeImages_v*", 
+        "E:\\docker\\windowsfilter\\*", 
+        "E:\\System\\Users\\*",
+        "\\\\?\\Volume{*}\\Windows\\*"
       )
     ) and
+    not process.executable : 
+              ("C:\\Packages\\Plugins\\Microsoft.GuestConfiguration.ConfigurationforWindows\\*\\dsc\\GC\\gc_*.exe", 
```

```suggestion
    not process.executable : (
```
This one seems to be limited to one env too